### DISCUSSION TITLE
Netative qrcode color

### DISCUSSION THREAD

**Discussion by Walbator** [mebjas/html5-qrcode#419] [Q&A]
Hi guys,

Thanks for that awesome code and tool. Love it and it worked like a charm.

I would like to know if there is a way for this to work scanning qrcode with opposit color ?

like : back black/dard and code white/clear ?

thank you

**Comment by ROBERT-MCDOWELL** [comment]
duplicate request, original here
https://github.com/mebjas/html5-qrcode/issues/101
